app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,6 @@
+module SessionsHelper
+  def auth_provider_path
+    strategy = Rails.env.development? ? 'developer_admin' : 'google_oauth2'
```

Comment:
I'd like to not use `Rails.env.development?` here (or in the other places). I'd rather say something like `Some.object.developer_admin?`, because there will be genuine cases where we want to use Google auth in dev.